R/helpers.R

```diff
@@ -215,3 +215,54 @@ create_dummy_parent_package <- function(mn, children) {
   list(parent = create_response,
        children = children)
 }
+
+
+#' Create dummy attributes data frame
+#'
+#' @param numberAttributes (integer) Number of attributes to be created in the table
+#' @param factors (character) Optional vector of factor names to include.
+#'
+#' @return dataframe
+#' @export
+#'
+#' @examples
+create_dummy_attributes_dataframe <- function(numberAttributes, factors=NULL) {
+  names <- sapply(1:numberAttributes, function(x){ paste0("Attribute ", x) })
```

Comment:
Try to always use `vapply` in favor of `sapply`. I'd rewrite this like:

```r
names <- vapply(seq_len(numberAttributes), function(x) { paste0("Attribute ", x)}, "")
```

`vapply` makes life much more sane because `sapply` isn't always guaranteed to return a character vector in this case but `vapply` (with its `FUN.VALUE` argument) is.

Other changes:


- `seq_len` is a safer version of `1:x`, a sibling to `seq_along`
- Added/removed whitespace to keep consistent styling